cuda-build.sh

```diff
@@ -38,7 +38,7 @@ echo "cuda_version ='"$CUDA_VERSION"'" >> build/awkward_cuda_kernels/__init__.py
 echo "docker_image ='docker.io/nvidia/cuda:"$DOCKER_IMAGE_TAG"'" >> build/awkward_cuda_kernels/__init__.py
 
 export DOCKER_ARGS="-v`pwd`:/home -w/home docker.io/nvidia/cuda:"$DOCKER_IMAGE_TAG
-export BUILD_SHARED_LIBRARY="nvcc -std=c++11 -Xcompiler -fPIC -Xcompiler -DVERSION_INFO="$AWKWARD_VERSION" -Iinclude src/cuda-kernels/*.cu --shared -o build/awkward_cuda_kernels/libawkward-cuda-kernels.so"
+export BUILD_SHARED_LIBRARY="nvcc -arch=sm_35 -std=c++11 -Xcompiler -fPIC -Xcompiler -DVERSION_INFO="$AWKWARD_VERSION" -Iinclude src/cuda-kernels/*.cu --shared -o build/awkward_cuda_kernels/libawkward-cuda-kernels.so"
```

Comment:
Should we revert this change?

Reply:
Hi! We were just on the moved Zoom call, talking about changes. This one had to be manually resolved. Why shouldn't this flag be included?

Reply:
Sorry I couldn't join. I am still in another meeting.

> This one had to be manually resolved. Why shouldn't this flag be included?

I thought we decided in an earlier meeting some months ago that we didn't want to explicitly state the `-arch` flag since we didn't get any direct benefit from specifying a lower bound of the target architecture.

Reply:
From follow up on Slack:

> I just mentioned it since I think we talked about not including it in some earlier meeting. I don't think there is any downside to keeping it since sm_35 has been supported by NVIDIA cards for quite a few years now.

> Okay. That might become an explicit requirement. I vaguely remember that conversation—I'm still unsure whether sm_35 is already implied by the CUDA >= 9 requirement.